tests/integration/test_messaging.rs

```diff
@@ -346,6 +347,72 @@ async fn can_deploy_l1_messaging_contract() {
     );
 }
 
+#[tokio::test]
+async fn setup_anvil_default_eth_private_key() {
+    let anvil = BackgroundAnvil::spawn().await.unwrap();
+    assert_eq!(
+        anvil.provider_signer.address(),
+        DEFAULT_ETH_ACCOUNT_ADDRESS.parse::<Address>().unwrap()
+    );
+}
+
+#[tokio::test]
+#[should_panic]
+async fn setup_anvil_incorrect_eth_private_key() {
+    let anvil =
+        BackgroundAnvil::spawn_with_additional_args(&[], Some(INCORRECT_ETH_ACCOUNT_PRIVATE_KEY))
+            .await
+            .unwrap();
+
+    let (devnet, _, _) = setup_devnet(&["--account-class", "cairo1"]).await;
+
+    let _body = devnet
+            .send_custom_rpc("devnet_postmanLoad", json!({ "network_url": anvil.url, "funded_account_private_key": INCORRECT_ETH_ACCOUNT_PRIVATE_KEY }))
+            .await
+            .expect("deploy l1 messaging contract failed");
+}
+
+#[tokio::test]
+async fn setup_anvil_correct_eth_private_key() {
```

Comment:
Tests `setup_anvil_incorrect_eth_private_key` and `setup_anvil_correct_eth_private_key` should be a single test.